include/OsqpEigen/SparseMatrixHelper.tpp

```diff
@@ -5,6 +5,8 @@
  * @date 2018
  */
 
+#include "Debug.hpp"
```

Comment:
Small thing. As before, could you please use the version with ``<>``

Reply:
Alright, done!